src/db.rs

```diff
@@ -7,7 +7,7 @@ use bytes::Bytes;
 
 // Temporarily, everything will be done in memory until we're happy with the
 // interface.
-#[derive(Debug)]
+#[derive(Debug, PartialEq, Eq)]
```

Comment:
There's something funny about deriving PartialEq and Eq on the database handle, since it's basically our entire in-memory db.  Totally the right call for now since it's temporary before the persistence layer, but it's funny.